.travis.yml

```diff
@@ -6,7 +6,7 @@ before_install:
 - sudo pip install -r test-requirements.txt
 
 script:
-- python app/tests/test.py
+- phantomjs app/tests/test.py
```

Comment:
Maybe this change is not required.

Reply:
Yes i do believe the same. 
@S2606 please look for it.

Reply:
phantomjs is a browser which can be used by many programming languages to test the code.
I do not think it understands the python file argument as running the file with python, more like visiting it as web page.